package.json

```diff
@@ -96,7 +96,7 @@
 					"name": "Cell Tags",
 					"type": "tree",
 					"icon": "$(tag)",
-					"when": "jupyter:showTagsExplorer",
+					"when": "jupyter:showTagsExplorer && jupyter.hasNativeNotebookOrInteractiveWindowOpen",
```

Comment:
wouldn't it be better to use some VS Code context keys like `notebookType == jupyter-notebook`
I personally think that would be better

Reply:
`notebookType` is a context key on editor groups which doesn't exist on panel context key tree unfortunately.